components/messenger.rst

```diff
@@ -282,23 +282,37 @@ do is to write your own CSV receiver::
             $this->filePath = $filePath;
         }
 
-        public function receive(callable $handler): void
+        public function get(): void
         {
             $ordersFromCsv = $this->serializer->deserialize(file_get_contents($this->filePath), 'csv');
 
             foreach ($ordersFromCsv as $orderFromCsv) {
                 $order = new NewOrder($orderFromCsv['id'], $orderFromCsv['account_id'], $orderFromCsv['amount']);
+                
+                $envelope = new Envelope($order);
 
-                $handler(new Envelope($order));
+                $handler($envelope);
```

Comment:
Is this right? I don't see how `$handler` can be defined here.

Reply:
Oh, nice one 🤔